Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
```

Comment:
Count me in for a strong 👎 on any strange/uncommon words like `Reactant`, `Reactivity`, `Reactiveness` because I feel they are forced adjectives. Reactive itself is an adjective, so there's no need to push it further.

I'll throw another few ideas out: `ReactiveExtensionsHosting`, `ReactiveExtensionsProtocol`, `ExtendedForReactiveSwift`, `ExposesReactiveBindings`, `ExposesReactiveExtensions`. I can go on all day… 😄